datamanager/splitLargeZip/splitZip.py

```diff
@@ -0,0 +1,113 @@
+#!/usr/bin/env python
+"""
+This script is used to split zip files (DM exports) that are above 1GB in size.
+
+Usage:
+    splitZip.py --path C:\\path\\to\\zip
```

Comment:
let's use snake_case for both variables and file names as this is Python standard:
https://www.python.org/dev/peps/pep-0008/
very short version: https://visualgit.readthedocs.io/en/latest/pages/naming_convention.html

Reply:
```suggestion
    split_zip.py --path C:\\path\\to\\import\\archive.zip
```